edeploy-lxc

```diff
@@ -65,12 +65,30 @@ def get_dist(rootfs):
         rel = 'centos'
     if os.path.isfile(dr_path) and file(dr_path).read().find('Debian GNU/Linux 7') == 0:
         rel = 'debian'
+    if os.path.isfile(rr_path) and file(rr_path).read().find('Fedora release') == 0:
+        rel = 'fedora'
     if rel:
         print("    detected target is %s" % rel)
         return rel
     raise Exception('Unsupported dist !')
 
-def stop_one(name, aufs_rw_dir):
+def get_union_filesystem(conf):
+    supported_fs = ['aufs', 'overlay']
+    if conf['edeploy'].has_key('union_fs'):
+        requested_fs = [conf['edeploy'].get('union_fs')]
+    else:
+        requested_fs = supported_fs
+    all_fs = [ l.split('\t')[-1] for l in open("/proc/filesystems").read().strip().split('\n') ]
```

Comment:
nitpicking, you can replace the `read().strip().split('\n')` this way:
[l.split('\t')[-1] for l in open('/proc/filesystems').read().splitlines()]